README.md

```diff
@@ -248,15 +253,39 @@ Notes:
 ```html
 <note class="os-teacher">
   <label>Teacher Edition</label>
-  <p>Students are usually bored by this but the simulation engages them. Try to jump to the sim first</p>
-    <list>
-      <item>(4C) <span class="ost-tag-teks-112-39-c-4c ost-teks-def">analyze and describe accelerated motion in two dimensions using equations, including projectile and circular examples</span></item>
+	<p>The Learning Objectives in this section will help your students master the following TEKS:</p>
+	  <list>
+	   <item>(4) Science concepts. The student knows and applies the laws governing motion in a variety of situations. The     
+	   student is expected to:
+	   </item>
+      <list>
+          <item class="ost-standards-def ost-standards-teks"><span class="ost-standards-name">(4C)</span>: <span   
```

Comment:
That's a question for the textbook folks. If they want to add special styling (`...:after { content: ": "; }`) then they'll need a non-`ost-` class name on the `ost-standards-name` element.